### DISCUSSION TITLE
Cannot download releases for private repository

### DISCUSSION THREAD

**Discussion by alexrutar** [community/community#8946] [Mobile] [answered]
Clicking the button to download the release files redirect to Safari, where the resource is reported as "Not found". I'm not logged into Github on Safari, which is likely causing this problem - however, I am logged into the Github app so it should be possible to access the file directly!

App version 1.44.0, iOS 15.1.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the report here @alexrutar — yep, I think that's the issue where you're signed out of the browser. Unfortunately we aren't able to authenticate the browser automatically even when you're signed into the app. The fix in the short term would be to sign into the website and tap the release download button again.
